File: dichtdelicht-python-controller/DTO/room.py
from DTO.LED import LED

class Room:
    def __init__(self, db_id, name, LEDs : [LED]):
        self.db_id = db_id
        
        self.name = name
        self.LEDs = LEDs

    def to_dict(self):
       return {
           u'name': self.name 
        }

    def __str__(self):
        LED_str = [str(led) for led in self.LEDs]
        return f"Room {{ name: '{self.name}', # LEDs: {len(self.LEDs)}  leds: {LED_str}  }}" 

File: dichtdelicht-ios/DichtDeLicht/Room.swift
//
//  Room.swift
//  DichtDeLicht
//
//  Created by Mariia Turchina on 09/05/2020.
//  Copyright © 2020 Mariia Turchina. All rights reserved.
//

import Foundation
import SwiftUI

struct Room : Hashable {
    var doc_id: String
    
    var name: String
    var LEDs: [LED]
}
